iPokeGo/es.lproj/Localizable.strings

```diff
@@ -0,0 +1,44 @@
+/* The title of the Pokémon selection for common Pokémon. */
+"Common" = "Común";
+
+/* The hint in a annotation callout that indicates when a Pokémon disappears. */
+"Disappears at" = "Desaparece a las %@";
+
+/* The title of the Pokémon selection for favorite Pokémon. */
+"Favorite" = "Favoritos";
+
+/* The title of a gym annotation on the map. */
+"Gym" = "Gimnasio";
+
+/* The description of a gym annotation on the map with points. */
+"Gym points: %d" = "Prestigio: %d";
+
+/* The message of an alert, that tells the user that he/she denied location access to the app. */
+"Location denied, please go in settings to allow this app to use your location" = "Ubicación desactivada. Abre Ajustes en el teléfono y permite a esta aplicación usar la ubicación";
+
+/* The title of an alert, that tells the user that he/she denied location access to the app. */
+"Location service denied" = "Ubicación desactivada";
+
+/* A common affirmative action title, like 'OK' in english. */
+"OK" = "OK";
+
+/* The message of an alert that tells the user, that no server was set. */
+"Please go in settings to enter server address" = "Dirijete a configuración para introducir la dirección del servidor";
```

Comment:
"Please go in settings to enter server address" = "Revise la configuración para introducir la dirección del servidor";

En cualquier caso, y manteniendo el 'Dirigete' (que no acaba de convencerme) habría que poner el acento en la í.

Un saludo y gracias por la traducción!